easybuild/easyblocks/p/pdt.py

```diff
@@ -57,36 +64,53 @@ def configure_step(self):
             toolchain.SYSTEM: '-GNU',
             toolchain.GCC: '-GNU',
             toolchain.INTELCOMP: '-icpc',
+            toolchain.PGI: '-pgCC',
         }
         comp_fam = self.toolchain.comp_family()
-        if comp_fam in known_compilers:
+        try:
             compiler_opt = known_compilers[comp_fam]
-        else:
+        except KeyError:
             raise EasyBuildError("Compiler family not supported yet: %s" % comp_fam)
         self.cfg.update('configopts', compiler_opt)
 
+        if self.toolchain.options['pic']:
+            self.cfg.update('-useropt=-fPIC')
+
+        # Configure creates required subfolders in installdir, so create first
+        super(EB_PDT, self).make_installdir()
```

Comment:
This call to `make_installdir` should be guarded, to ensure it's only done once in case of an iterated installation (for example when the `configopts` specified in the PDT easyconfig file is a list of string values, rather than a single string value).
The previous approach via `keeppreviousinstall` did that sort of too, but caused problems with a forced reinstallation, so this approach is better as long as the guard is there:

```python
# create installation directory (but only once, during first iteration)
if self.iter_idx == 0:
    super(EB_PDT, self).make_installdir()

Reply:
Done here and in TAU